go.mod

```diff
@@ -279,6 +287,7 @@ require (
 
 replace (
 	github.com/onsi/ginkgo/v2 => github.com/openshift/onsi-ginkgo/v2 v2.6.1-0.20230317131656-c62d9de5a460
+	go.opentelemetry.io/otel/metric => go.opentelemetry.io/otel/metric v0.38.1
```

Comment:
pPointed this to the latest version thta statifies the issues stemming from .

Reply:
Mentioning here that the parent PR is free of these dependency conflicts, which come into play once that's rebased on `master`.

Reply:
https://github.com/open-telemetry/opentelemetry-go/issues/4255

Reply:
`go vet ./...` and `./hack/update-generated` passing now.

Reply:
`go vet ./...` and `./hack/update-generated.sh` passing now.